external/webrtc-sdk.js

```diff
@@ -32691,9 +32691,17 @@ define("WebRTC_SDK/manager/RTCLocalMediaManager", [
       var deferred = Q.defer();
```

Comment:
GitHub is not letting me expand this file as it's too big. This is the function body to `muteLocalAudio` which is called by `toggleMuteAudio`

Reply:
```js
    muteLocalAudio: function (mute) {
      var deferred = Q.defer();

      var stream = this.model.get("localAudioStream");
      if (_.isNull(stream) || _.isUndefined(stream)) {
        Logger.warn(
          "Local Audio stream is null, only mute/unmute local backbone model."
        );
        this.model.set("localAudioMuted", mute);
        deferred.resolve();
        return deferred.promise;
      }
      ...
```